src/Libraries/CoreNodes/Files.cs

```diff
@@ -97,7 +97,7 @@ public static string ReadText(FileInfo file)
         }
 
         /// <summary>
-        /// 
+        ///  Move a file to a new Path
```

Comment:
Moves a specified file to a new location
You can look at documentation of System.IO.File.Move to get the correct wording.